src/www/base.webpack.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const WebpackShellPluginNext = require('webpack-shell-plugin-next');
+
+const OUTPUT_BASE = path.resolve(__dirname, '../../www');
+
+exports.makeConfig = (options) => {
+  return {
+    entry: [
+      require.resolve('@webcomponents/webcomponentsjs/webcomponents-bundle.js'),
+      path.resolve(__dirname, './style.css'),
+      // path.resolve(__dirname, './ui_components/licesnses/licenses.txt'),
+      options.main,
+    ],
+    mode: 'production',
```

Comment:
Should pass this as an option, or is it automatically applied with the `--mode` flag?

Reply:
So the mode flag does work and that's how I've been changing it.

export BUILD_ENV=development
yarn do src/electron/run